feat: add ability to use ga events for analytics (#3939)
electron/electronjs.org: CONTRIBUTING.md

- [Context](#context)
- [Styles](#styles)
- [Frontend JavaScript](#frontend-javascript)

> - [GA Evens](#google-analytics-events)

Reviewer: ```suggestion
- [Google Analytics Events](#google-analytics-events)
```

disk. When a GET request is made to `/scripts/index.js`, the server dynamically
compiles the `scripts/index.js` file to browser-compatible ES5.

## Google Analytics Events

> We use the [Google Analytics Events](https://developers.google.com/analytics/devguides/collection/analyticsjs/events)
> for measure purposes of needing and usage of features what are we
> provide on the website.
> 
> For usage, you need to provide the `data-sc-category` and `data-sc-name` properties
> to element. If `data-sc-category` is not provided `general` be used as a
> default category for event.

Reviewer: ```suggestion
We use [Google Analytics Events](https://developers.google.com/analytics/devguides/collection/analyticsjs/events)
to measure usage of features that we provide on the website.

To add an event, you need to provide `data-sc-category` and `data-sc-name`
properties to an element in the DOM. If `data-sc-category` is not provided,
`general` will be used as a default category for the event.
```
